FrostyEditor/Themes/UserControls/TitleBar.axaml

```diff
@@ -0,0 +1,15 @@
+<UserControl xmlns="https://github.com/avaloniaui"
```

Comment:
this [does not work on linux](https://docs.avaloniaui.net/docs/next/tutorials/music-store-app/creating-a-modern-looking-window#acrylic-blur), so i think just setting the title of the window is enough for now

Reply:
I didn't set the acrylic blur and the user-control should works
If only set title it will use mica on windows11 and looks worse